plugins/cisco_asa/plugin.spec.yaml

```
@@ -224,3 +250,12 @@ actions:
         description: Returns true if object was created
         type: boolean
         required: true
+  get_blocked_hosts:
+    title: Get Blocked Hosts
+    description: Get hosts blocked with shun command
+    output:
+      hosts:
+        title: Hosts
+        description: List of hosts blocked with shun command
+        type: "[]hosts"
+        required: false 
```

```suggestion
        required: true
```
Make sure that we always return output even if there are 0 elements in the array